boefjes/boefjes/sql/db_models.py

```diff
@@ -14,6 +15,36 @@ class ScanLevel(Enum):
     L4 = 4
 
 
+class RunOnDB(Enum):
+    CREATE = "create"
+    UPDATE = "update"
+    CREATE_UPDATE = "create_update"
+
+    @classmethod
+    def from_run_ons(cls, run_ons: list[RunOn] | None):
```

Comment:
```suggestion
    def from_run_ons(cls, run_ons: list[RunOn] | None) -> RunOn | None:
```

Reply:
@madelondohmen This should be `Self` from typing_extensions (probably) it should actually return `RunOnDB | None` which won't work because it references itself. 
```suggestion
    def from_run_ons(cls, run_ons: list[RunOn] | None) -> Self | None:
```

Reply:
`typing.Self` is introduced in Python 3.11, but we still use 3.10 for development. Therefore:
`def from_run_ons(cls, run_ons: list[RunOn] | None) -> "RunOnDB" | None:`